src/software/ai/hl/stp/play/shoot_or_chip_play.cpp

```diff
@@ -28,8 +28,11 @@ std::string ShootOrChipPlay::getName() const
 
 bool ShootOrChipPlay::isApplicable(const World &world) const
 {
-    return world.gameState().isPlaying() &&
-           teamHasPossession(world, world.friendlyTeam());
+    // TODO: figure out whether or not ShootOrChip and ShootOrPass should be separate
```

Comment:
Thoughts on this? I think they should be combined (perhaps in a future PR when we get to more strategy changes).

The main differences are that ShootOrChip will chip instead of pass when the robot with the ball is threatened by an enemy, and it uses findOpenAreas instead of cherry pickers to move robots into space.

Reply:
We could just leave this as:
```
// NOTE: We do not currently use this play, as passing is generally superior. However we keep it around and
//              maintain it so as to have a backup if passing become unreliable for whatever reason
return true;
```
but I'm definitely on the fence about this one. If we keep it around we should make sure to have simulated tests for it to stop regressions.